scripts/create_sample_meta.py

```diff
@@ -33,10 +33,28 @@ def save_dataframe(df, build_dir):
 
 
 def rename_columns(df):
-    rename_map = {"index1": "index_1", "index2": "index_2", "pert_id": "compound_id"}
+    rename_map = {"index1": "index_1", "index2": "index_2", "trt_type": "pert_type",
```

Comment:
@jdavis3141 will we need to get or rename prc_id explicitly here?
Or will prc_id now be passed as prc_id by the COMET view changes  made by @AnupJonchhe ?

Reply:
Right now the prc_id column from the UI resolves to pert_id when querying the api, so if we want to use prc_id we will need to rename it.

Reply:
Leaving this alone for now until changes are made to the view